Model/getPosts.php

```diff
@@ -0,0 +1,164 @@
+
+<script>
+
+    $(document).ready(function () {
+
+        // To initially set all counts of liks
+        $('.like-count').each(function () {
+            $userId = ($(this).parent().parent().parent().attr('id').split('-')[0]);
+            $postId = $(this).attr('id').substr(11);
+            $(this).load("../Model/updateLike.php?postId=" + $postId + " #totalLikes");
+        });
+
+        // To update status of likes
+        $.get("../Model/updateLike.php?&userId=" + $userId, function ($r) {
+            $.each($r.split(" "), function ($key, $val) {
+                if ($val != "") {
+                    $("#like-" + $val).addClass('liked');
+                    $("#like-" + $val).attr("liked", "TRUE");
+                }
+            });
+
+        });
+
+        // To update like counts in DB
+        $('.like').click(function () {
+            $(this).toggleClass('liked');
+            $userId = $(this).parent().parent().attr('id').split('-')[0];
+            ($(this).attr("liked") == 'FALSE') ? $(this).attr("liked", 'TRUE') : $(this).attr("liked", 'FALSE');
+            $liked = $(this).attr("liked");
+            $postId = $(this).attr('id').substr(5);
+            $likeCount = $(this).children('.like-count').children('#totalLikes').html();
+            if ($liked == 'TRUE')
+                $(this).children('.like-count').children('#totalLikes').html(parseInt($likeCount) + 1);
+            else
+                $(this).children('.like-count').children('#totalLikes').html(parseInt($likeCount) - 1);
+            $likeCount = $(this).children('.like-count').children('#totalLikes').html();
+            $.get("../Model/updateLike.php?pi=" + $postId + "&ui=" + $userId + "&li=" + $liked, function (data, status) {
+            });
+
+        });
+
+        // To show comment section
+        $('.comment-btn').click(function () {
+            $commentsSection = $(this).parent().parent().children('.comments-div');
+            $userId = $commentsSection.children('.comments').parent().parent().attr('id').split('-')[0];
+            $postId = $commentsSection.children('.comments').parent().parent().attr('id').split('-')[2];
+            $commentsSection.children('.comments').load("../Model/updateComment.php?pi=" + $postId + "&ui=" + $userId, function (r) {
+            });
+            $commentsSection.slideToggle();
+        });
+
+
+        ///////MENU BUTTON///////
```

Comment:
try to write proper comments.
`//  Menu button`

Reply:
okay